examples/app-basic/tests/fixtures.ts

```diff
@@ -14,6 +17,13 @@ export const test = base.extend<AppTestFixture>({
     await configPage.goto();
     await use(configPage);
   },
+  gotoPage: ({ gotoAppPage }, use) =>
+    use((path) =>
+      gotoAppPage({
```

Comment:
Need to return our await this invocation

Reply:
It does return since I don't have any `{}` around the `gotoPage` function. But I can wrap it and return if it makes it more explicit.